wire-ios/Wire-iOS/Sources/Analytics/TrackingManager/TrackingManager.swift

```diff
@@ -21,6 +21,10 @@ import Foundation
 import WireCommonComponents
 import WireSyncEngine
 
+enum TrackingMAnagerError: Error {
```

Comment:
```suggestion
enum TrackingManagerError: Error {
```

Would be nice to also move to a new file.

Reply:
Done!